projects/plugins/jetpack/extensions/blocks/tiled-gallery/tiled-gallery.php

```diff
@@ -46,6 +46,26 @@ public static function register() {
 		}
 	}
 
+	/**
+	 * Registers inline scripts which populate `window.JetpackTiledGalleryOptions` in Javascript.
+	 *   window.JetpackTiledGalleryOptions.isAtomicSite (Boolean)  - Is the site atomic?
+	 *   window.JetpackTiledGalleryOptions.isPrivateSite (Boolean) - Is the site private?
+	 *
+	 * @return void
+	 **/
+	public static function inject_javascript_options() {
+		$options = array(
+			// 'isAtomicSite'  => jetpack_is_atomic_site(),
+			// ################################### WARNING #####################################.
+			'isAtomicSite'  => true, // XXX TODO DEBUG DO NOT MERGE.
+			// ################################### WARNING #####################################.
+			'isPrivateSite' => '-1' === get_option( 'blog_public' ),
+		);
+		wp_enqueue_script( 'tiled-gallery-options', '', array(), 'v0.0-inline', true );
+		// Use wp_add_inline_script instead of wp_localize_script, see https://core.trac.wordpress.org/ticket/25280.
+		wp_add_inline_script( 'tiled-gallery-options', 'var JetpackTiledGalleryOptions = JSON.parse( decodeURIComponent( "' . rawurlencode( wp_json_encode( $options ) ) . '" ) );' );
```

Comment:
This data is general data pertaining to Atomic sites (and we have multiple galleries and blocks that could use it) – maybe `JetpackWoAData` instead?..

Reply:
Changed here: b3da63c00